### DISCUSSION TITLE
fee distribution

### DISCUSSION THREAD

**Discussion by techOptimizor** [Jeiwan/uniswapv3-book#17] [Milestone 5. Fees and Price Oracle] [answered]
How do uniswap calculate and distribute fee to liquidity providers

**Comment by Jeiwan** [comment, marked as answer]
There's a chapter on that:
https://uniswapv3book.com/docs/milestone_5/swap-fees/

**Reply by techOptimizor** [reply]
I just understood that .
Thank you.
